src/web/src/pages/CourseExplorer.vue

```diff
@@ -148,14 +161,35 @@ export default {
     },
   },
   computed: {
-    //Used to define the placement of each department into the two main columns
+    /**
+     *coursesChunked
+     * Used to define the placement of each department into the two main columns
+     * Data structure -> An Array contains 6 list, each list contains one string
+     * @returns {string[][]} courses grouped by visual blocks
+     */
     coursesChunked() {
       const chunkedArr = [];
-      for (var i = 0; i < 6; i++) {
-        chunkedArr.push(this.schoolsMajorDict[this.schoolOrder[i]]);
+      let majorsArr = Object.values(this.schoolsMajorDict);
+      majorsArr.sort((a, b) => b.size - a.size);
+      for (let i = 0; i < 6; i++) {
+        chunkedArr.push(majorsArr[i]);
       }
       return chunkedArr;
     },
+
+    schoolOrder() {
```

Comment:
An alternative approach to having two computed methods `coursesChunked` and `schoolOrder` is to have a single computed method that **returns an array of objects**, each object containing a school and its majors. 

So something like this:
```javascript
computed: {
  schoolDepartmentObjects () {
    // Object.entries returns a list of key-value pairs
    return Object.entries(this.schoolsMajorDict)
                   // convert the lists into an object so they are easier to use
                   // [school, majors] -> {school, majors}
                   .map((schoolDepartmentMapping) => ({
                      school: schoolDepartmentMapping[0],
                      departments: schoolDepartmentMapping[1],
                   })
                   // sort objects by size of departments
                   .sort((left, right) => right.departments.size - left.departments.size); 
  }
}
```

This way, in the html, instead of doing a`v-for` on a number range and then calling `{{ schoolOrders[n-1] }}`, you can do something like

```vue
<div v-for="schoolDepartmentObj in schoolDepartmentObjects" :key="schoolDepartmentObj.school">
  ...
  {{ schoolDepartmentObj.school }}
  ...
  <div v-for="department in schoolDepartmentObj.departments">
    {{ department }}
  </div>
  ...
</div>
```

Generally, try to use arrays of objects and iterate over objects instead of using the indexes